caffe2/CMakeLists.txt

```diff
@@ -1343,6 +1355,16 @@ if(USE_DISTRIBUTED)
   if(USE_GLOO AND USE_C10D_GLOO)
     target_compile_definitions(torch_cpu PUBLIC USE_C10D_GLOO)
   endif()
+  if(USE_UCC AND USE_C10D_UCC)
+    target_compile_definitions(torch_cpu PUBLIC USE_C10D_UCC)
```

Comment:
Why does it have to be a public definition? Should consumers of libtorch_cpu be aware about this one. If they do, we better put it in  the autogenerated TorchConfig.h

Reply:
Mainly for consistency with other target_compile_definitions such as USE_C10D_GLOO, USE_C10D_NCCL, USE_C10D_MPI etc.